### ISSUE TITLE
Collection Object Attribute duplicated in permissions system

### ISSUE THREAD

**Issue by mpitblado** [specify/specify7#5407]
**Describe the bug**
When selecting which tables a user has access to, 'Collection Object Attribute' appears twice. One of which will refer to Collectionobjectattr while the other will refer to Collectionobjectattribute, but the user setting the permission has no way of knowing which is which.

**To Reproduce**

1. Go to Security and Accounts
2. Select a Collection
3. Under "Custom Collection Level Policies" select `Table`
4. Notice that `Collection Object Attribute` appears twice.

**Expected behavior**
The two tables would not be referred to in the same way to allow differentiation. Collection Object Atr for one and Collection Object Attribute for the other would align with the API documentation.

**Screenshots**
![image](https://github.com/user-attachments/assets/35bb6a2b-9a28-4c77-9c2c-49bb92fa8be8)

**Crash Report**
N/A

Please fill out the following information manually:
- Specify 7 Version: 7.9.6.2

**Reported By**
Beaty Biodiversity Museum

**Additional context**
In the [schema](https://files.specifysoftware.org/schema/version/2.10/#collectionobjectattribute), the tables are also titled the same thing, however the difference between them can be seen in the url's.

**Comment by grantfitzsimmons** [comment]
Hi @mpitblado,

This is actually because we have _two_ Collection Object Attribute tables– both with the same table name assigned in the interface schema, which is what you see on the Security and Accounts list of tables.

You can see this in the database directly, and the table names are:
- `collectionobjectattribute`
   - This is a one-to-one table that captures all the additional attributes of a CO that do not belong elsewhere. It is the most commonly used table between the two and is included in the default form for most disciplines.
- `collectionobjectattr`
   - This was intended to be a one-to-many 'Collection Object Property'-like relationship to enable infinite attributes for an object (back in 2008), but we have yet to revisit or use this table. 

Since both tables are technically usable in Specify (you can add both to your form!), they remain independent with the same name. You can change the table name in the schema to reduce confusion!

**Comment by mpitblado** [comment]
Hi @grantfitzsimmons,

Makes sense, but should the default names for them be different (I don't think I changed the caption in the schema for either but I might be mistaken in that). In using the forms in the api and in the xml for forms, they have the names `collectionobjectattribute` and `collectionobjectattr` to distinguish them, no?

**Comment by grantfitzsimmons** [comment]
Yes, definitely! Out of the box it duplicates the name in the schema. This is something we should change in the future to resolve this issue.
